From debe2a6ac3fae48a9aaa1aef0c94decf8dd45d18 Mon Sep 17 00:00:00 2001
From: Tony Camuso <tcamuso@redhat.com>
Date: Tue, 18 Oct 2016 08:04:02 -0400
Subject: MANPATH should not be used as install dir

installation fails when MANPATH variable is defined with more than one
directory.

make -C docs install
make[2]: Entering directory `/home/test/rpmbuild/BUILD/tboot-1.8.3/docs'
[ -d /usr/share/man/mpich:/usr/share/man/man8 ] || install -d -m0755 -p /usr/share/man/mpich:/usr/share/man/man8
install: cannot create directory `/usr/share/man/mpich:': Permission denied
make[2]: *** [install] Error 1

Man files should be installed into DISTDIR not into '/' of system.

from manpage of man:
MANPATH
     If $MANPATH is set, its value is used as the path to search for
     manual pages.
---
 docs/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/docs/Makefile b/docs/Makefile
index 37869d1..953aa9e 100644
--- a/docs/Makefile
+++ b/docs/Makefile
@@ -11,7 +11,7 @@ ROOTDIR ?= $(CURDIR)/..
 
 include $(ROOTDIR)/Config.mk
 
-MANPATH ?= $(DISTDIR)/usr/share/man
+TBOOT_MANPATH ?= $(DISTDIR)/usr/share/man
 
 
 #
@@ -26,8 +26,8 @@ dist : install
 
 
 install :
-	[ -d $(MANPATH)/man8 ] || $(INSTALL_DIR) $(MANPATH)/man8
-	$(INSTALL_DATA) -t $(MANPATH)/man8 man/*.8*
+	[ -d $(TBOOT_MANPATH)/man8 ] || $(INSTALL_DIR) $(TBOOT_MANPATH)/man8
+	$(INSTALL_DATA) -t $(TBOOT_MANPATH)/man8 man/*.8*
 
 
 clean :
-- 
1.8.3.1

